exonum/build.rs

```diff
@@ -12,10 +12,23 @@ fn create_path_to_protobuf_schema_env() {
     // Workaround for https://github.com/rust-lang/cargo/issues/3544
     // We "link" exonum with exonum_protobuf library
     // and dependents in their `build.rs` will have access to `$DEP_EXONUM_PROTOBUF_PROTOS`.
-    let path = env::current_dir()
-        .expect("Failed to get current dir.")
-        .join("src/proto/schema/exonum");
-    println!("cargo:protos={}", path.to_str().unwrap());
+
+    let current_dir = env::current_dir().expect("Failed to get current dir.");
+
+    let protos = current_dir.join("src/proto/schema/exonum");
+    println!("cargo:protos={}", protos.to_str().unwrap());
+
+    let crypto_protos = current_dir
+        .join("../components/crypto/src/proto/schema")
```

Comment:
This won't work if the `exonum` crate is downloaded from crates.io. It seems that it is necessary to perform Protobuf generation for each crate separately.

Reply:
Is it possible to test this case without publishing?

Reply:
[`cargo package`](https://doc.rust-lang.org/cargo/reference/publishing.html#packaging-a-crate)?